img/ADBC.svg

```diff
@@ -0,0 +1 @@
trix(0.06172839506172839,0,0,0.06172839506172839,30.864197530864196,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,43.20987654320987,0)" xlink:href="#aF"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,55.55555555555555,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,67.90123456790123,0)" xlink:href="#ax"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,79.01234567901234,0)" xlink:href="#am"/></g><g id="x"><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,0,0)" xlink:href="#ay"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,14.814814814814813,0)" xlink:href="#av"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,22.160493827160494,0)" xlink:href="#av"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,29.50617283950617,0)" xlink:href="#as"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,41.85185185185185,0)" xlink:href="#au"/></g><g id="y"><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,0,0)" xlink:href="#az"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,12.345679012345679,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,24.691358024691358,0)" xlink:href="#ar"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,30.864197530864196,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,43.20987654320987,0)" xlink:href="#aB"/></g><g id="z"><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,0,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,12.345679012345679,0)" xlink:href="#an"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,23.45679012345679,0)" xlink:href="#ar"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,29.62962962962963,0)" xlink:href="#aE"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,41.97530864197531,0)" xlink:href="#aA"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,54.32098765432099,0)" xlink:href="#ao"/></g><g id="A"><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,0,0)" xlink:href="#an"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,11.11111111111111,0)" xlink:href="#ao"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,15.987654320987653,0)" xlink:href="#ap"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,20.864197530864196,0)" xlink:href="#am"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,33.20987654320987,0)" xlink:href="#aq"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,45.55555555555555,0)" xlink:href="#ar"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,51.72839506172839,0)" xlink:href="#aG"/></g></defs></g></svg>
```

Comment:
This text in the diagram seems confusing:
>The database only works with Arrow data, regardless of the actual client.

The database does not necessarily "work with" Arrow data. It (ultimately) emits Arrow data, but internally might be working with the data in some other database-specific format. Is there a better way to express what you mean by this?

Reply:
I meant something like that, yeah. The database only has to implement one (columnar, Arrow-based) endpoint/protocol but can support Arrow-native and 'traditional' clients.